ddocs/medic/_attachments/translations/messages-en.properties

```diff
@@ -1385,6 +1387,32 @@ user.place.help = All users must be associated with a place in the system to be
 user.username.help = This is what you will use to log in to the app.
 username.invalid = Invalid user name. Valid characters are lower case letters, numbers, underscore (_), and hyphen (-).
 username.taken = Username "{{username}}" already taken.
+users.import.accepted_types = Only .csv file are accepted.
+users.import.cancel_upload = Cancel
+users.import.confirm_upload = Yes, upload
+users.import.confirm_upload_text = Are you sure you want to upload
+users.import.instructions = To add multiple users, please upload the appropriate spreadsheet.
+users.import.processing.title = Processing users
+users.import.processing.instructions = This process can take a long time, if you're uploading a lot of users, please be patient.
+users.import.summary.added_users = users added
+users.import.summary.click = Click
+users.import.summary.download_status_file = to download your status file.
+users.import.summary.failed_users = contained errors
+users.import.summary.here = here
+users.import.summary.previously_added_users = previously imported
```

Comment:
This one after
`users.import.summary.resubmit.instructions`

Reply:
On this I was sorting by the first letter after `summary` so it made sense to have it in this order